project/DistributionPackage.scala

```diff
@@ -6,6 +6,29 @@ import sbt.util.{CacheStore, CacheStoreFactory, FileInfo, Tracked}
 import scala.sys.process._
 
 object DistributionPackage {
+
+  /** File extensions. */
+  implicit class FileExtensions(file: File) {
+
+    /** Get the outermost directory of this file.
+      *
+      * == Example ==
+      * {{{
+      *   file("foo/bar/baz").getTopDirectory == file("foo")
+      * }}}
+      *
+      * @return the outermost directory of this file.
+      */
```

Comment:
You should note that this will resolve to root `/` if the file is an absolute path.

Reply:
That's expected behavior for the absolute path. I'll mention that in the comment.